polkadot/runtime/parachains/src/coretime/mod.rs

```diff
@@ -281,3 +364,55 @@ fn mk_coretime_call<T: Config>(call: crate::coretime::CoretimeCalls) -> Instruct
 		call: BrokerRuntimePallets::Broker(call).encode().into(),
 	}
 }
+
+fn do_notify_revenue<T: Config>(when: BlockNumber, raw_revenue: Balance) -> Result<(), XcmError> {
```

Comment:
Do we have a test for this?

Reply:
We have a WIP test for everything not yet merged in but already working: https://github.com/s0me0ne-unkn0wn/coretime-revenue-test/blob/master/src/main.rs

Do you think a separate unit test could be useful here?

Reply:
Yes

Reply:
Raised a follow-up: #4882